src/ray/gcs/gcs_server/gcs_actor_scheduler.cc

```diff
@@ -123,6 +123,15 @@ std::vector<ActorID> GcsActorScheduler::CancelOnNode(const ClientID &node_id) {
   return actor_ids;
 }
 
+void GcsActorScheduler::CancelOnLeasing(const ClientID &node_id,
+                                        const ActorID &actor_id) {
+  /// Note(sang): It is currently effective only inside RequestWorkerLease callback.
+  /// TODO(sang): Cancel lease request explictly from this API.
```

Comment:
What does this comment mean? What API do you mean?

Reply:
This means to call `raylet_client->CancelLeaseRequest`. I will also modify this comment.